spanner/src/list_backup_operations.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/spanner/README.md
+ */
+
+namespace Google\Cloud\Samples\Spanner;
+
+// [START spanner_list_backup_operations]
+use Google\Cloud\Spanner\SpannerClient;
+
+/**
+ * List all create backup operations in an instance.
+ * Example:
+ * ```
+ * list_backup_operations($instanceId, $databaseId);
+ * ```
+ *
+ * @param string $instanceId The Spanner instance ID.
+ * @param string $databaseId The Spanner database ID.
+ */
+function list_backup_operations($instanceId, $databaseId)
+{
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+
+    // List the CreateBackup operations.
+    $filter = "(metadata.database:$databaseId) AND " .
+              "(metadata.@type:type.googleapis.com/" .
+              "google.spanner.admin.database.v1.CreateBackupMetadata)";
+
+    $operations = $instance->backupOperations(['filter' => $filter]);
+
+    foreach ($operations as $operation) {
+        if (!$operation->done()) {
+            $operationName = basename($operation->info()['name']);
+            print("Backup operation $operationName pending" . PHP_EOL);
```

Comment:
```suggestion
            print("Backup $metadataName on database $metadataDatabase is $progressPercent% pending" . PHP_EOL);
```
I'm not sure how to get the metadata out of the operation in PHP but all the necessary info should be in there like the name, database and progress_percent.

Reply:
Current implementation returns raw, binary-encoded metadata as part of operation info. I don't know how to decode it. Looks like an oversight to me, but this implementation is approved.

Reply:
@AVaksman would you be able to help with this? The reason why we wanted to do this was to show users through the sample how to decode the metadata because it's not obvious. I don't know how it would work in PHP but it would be good to make this consistent with that of the other languages. E.g. in [Node.js](https://github.com/googleapis/nodejs-spanner/blob/e3e454dd9484d70acada78a1ab1feac68346ebac/samples/backups-get-database-operations.js#L45).

Reply:
Friendly ping on this comment. We need the samples to be consistent across the languages before they can be merged. We have to get this submitted within the next couple of days.

Reply:
Couple of days? That will be hard.
@AVaksman proposes ([#2536](https://github.com/googleapis/google-cloud-php/pull/2536)) some changes inside Core to solve this problem, but that solution has problems itself.

Reply:
Wow ok, I understand why it's hard then. Thanks for the background info. I'm happy to approve this as is but I also filed #1050 to fix the samples once you're able to decode the metadata.